wfsim/strax_interface.py

```diff
@@ -593,8 +593,7 @@ def get_instructions(self):
             # Why epix removes many of the g4ids?
             # Remove nveto event if no tpc event of the same g4id is found
             if len(self.g4id) > 0:
-                nv_inst_to_keep = np.isin(self.instructions_nveto['g4id'], self.g4id[0])
-                nv_inst_to_keep &= (self.instructions_nveto['_last'] - self.instructions_nveto['_first']) > 0
+                nv_inst_to_keep = (self.instructions_nveto['_last'] - self.instructions_nveto['_first']) >= 0
             self.instructions_nveto = self.instructions_nveto[nv_inst_to_keep]
             self.g4id.append(self.instructions_nveto['g4id'])
             log.debug("Epix produced %d instructions in nv" % (len(self.instructions_nveto)))
```

Comment:
epix is not involved here, I'd change this string.

Reply:
I agree with you.

Reply:
#185